### PR TITLE
Support auto registe when storage start up

### PR DESCRIPTION
**Pull request by darionyaphet**
#### What type of PR is this?
- [ ] bug
- [ ] feature
- [ ] enhancement

#### Checklist:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the corresponding label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

#### Release notes:

Please confirm whether to be reflected in release notes and how to describe:
>                                                                 `

### CODE REVIEW

**Review by critical27 — COMMENTED** [review]
Maybe reuse the old parameter name `hosts_whitelist_enabled`

File: src/meta/processors/admin/HBProcessor.cpp [hunk 1]
```diff
@@ -34,7 +36,7 @@ void HBProcessor::process(const cpp2::HBReq& req) {
             << ", role = " << apache::thrift::util::enumNameSafe(role);
 
   if (role == cpp2::HostRole::STORAGE) {
-    if (!ActiveHostsMan::machineRegisted(kvstore_, host)) {
+    if (!FLAGS_auto_registe && !ActiveHostsMan::machineRegisted(kvstore_, host)) {
```
Comment (panda-sheep) [inline src/meta/processors/admin/HBProcessor.cpp:39]: I have a problem
For other roles such as storage_listener, do I need to add hosts?

Comment (panda-sheep) [reply src/meta/processors/admin/HBProcessor.cpp:39]: In my test, other roles are not cpp2::HostRole::STORAGE, add host succeeds, but show does not come out

Comment (darionyaphet) [reply src/meta/processors/admin/HBProcessor.cpp:39]: https://github.com/vesoft-inc/nebula/pull/3444  have fix it

Comment (panda-sheep) [reply src/meta/processors/admin/HBProcessor.cpp:39]: I think you should merge that bug in first, otherwise, the entire cluster will be useless.

**Review by jievince — APPROVED** [review]

**Review by Shylock-Hg — APPROVED** [review]

**Review by jievince — APPROVED** [review]

**Pull request merged**